test-suites/src/moderc3156.rs

```diff
@@ -0,0 +1,17 @@
+use soroban_sdk::{testutils::Address as _, Address, Env};
+
+mod emitter_contract {
+    soroban_sdk::contractimport!(file = "../target/wasm32-unknown-unknown/optimized/moderc3156_example.wasm");
```

Comment:
we need to add this to the `Makefile` so that the `moderc3156_example.wasm` is built during `make build`, such that things like the github test pipeline can access it.